.github/workflows/checks.yml

```diff
@@ -21,7 +21,7 @@ jobs:
 
       - name: Remove files irrelevant files that may cause TypeScript errors
         run:
-          rm -f ./seed.mts
+          rm -f supabase/seed/seed.mts
```

Comment:
We should do this in other places too (other workflows & readme). Could you ctr + shift + f `seed.mts` please?